tests/integration/extenders/ViewTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Tests\integration\extenders;
+
+use Flarum\Extend;
+use Flarum\Tests\integration\TestCase;
+use Illuminate\Contracts\View\Factory;
+
+class ViewTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function custom_view_namespace_does_not_exist_by_default()
+    {
+        $this->expectException(\InvalidArgumentException::class);
+        $this->assertEquals('<html><body>Hello World!</body></html>', trim($this->app()->getContainer()->make(Factory::class)->make('integration.test::test')->render()));
```

Comment:
I assume you're going for symmetry with the test case below here, but it confuses me that there is an assertion after the `expectException()` call. That primarily signals to me that we expect this assertion to hold true.

Which call here is going to throw the `InvalidArgumentException`? Is it `Factory::make()`? In that case, let's only call this method, please. Or is it the `assertEquals()`? (I would assume that throws a custom exception class.)

If you want more symmetry, you could do the `$this->app()...->render()` on one line in both tests, and in the second one store the result in a variable and then `assertEquals()` on that in an extra line.